frontends/web/src/components/groupedaccountselector/groupedaccountselector.module.css

```
@@ -162,12 +186,52 @@
+}
 
 .placeholderText {
     flex: 1;
     text-align: left;
 }
 
+.stackedSelect :global(.react-select__control),
+.stackedSelect :global(.react-select__control:hover) {
+    background: transparent;
+    border-color: transparent;
+    border: none;
```

Stylelint (17.7.0)

[error] Overridden property "border-color" by shorthand "border" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)